src/ComponentWrapper.js

```diff
@@ -10,7 +10,7 @@ class ComponentWrapper {
     checkStatusInterval = 5 /*minutes*/ * 60000,
     debug = container.debug,
     noColors = container.noColors
-  }) {
+  }, options = {}) {
```

Comment:
I don't like to use a different object for options.
I suggest to merge standard options with component specific options.
In order to do this I suggest to have an .options property into the Component class instead of keeping all standard options as single properties

Reply:
Good idea, I'm on it.